### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by phuocbui31** [Pandede/WPODNet-Pytorch#20]
Thank you for your source code.
Can you help me about loss function and how to train model in pytorch ?

**Comment by Pandede** [comment]
I'm still seeking for the training procedure and loss function either.
Welcome to create the PR if you've any idea or implementation about training!

**Comment by phuocbui31** [comment]
Thank you for your reply. I am learning from your project. However, I don't know how to implement the loss function and the training procedure. If you have, please guide me!
